controller/search.go

```diff
@@ -49,9 +50,39 @@ func (c *SearchController) Show(ctx *app.ShowSearchContext) error {
 	urlRegexString = fmt.Sprintf("(?P<domain>%s)(?P<path>/work-item/board/detail/)(?P<id>\\d*)", hostString)
 	search.RegisterAsKnownURL(search.HostRegistrationKeyForBoardWI, urlRegexString)
 
+	if ctx.FilterExpression != nil {
+		return application.Transactional(c.db, func(appl application.Application) error {
+
+			result, c, err := appl.SearchItems().Filter(ctx.Context, *ctx.FilterExpression, &offset, &limit)
+			count := int(c)
+			if err != nil {
+				cause := errs.Cause(err)
+				switch cause.(type) {
+				case errors.BadParameterError:
+					jerrors, _ := jsonapi.ErrorToJSONAPIErrors(goa.ErrBadRequest(fmt.Sprintf("error listing work items: %s", err.Error())))
+					return ctx.BadRequest(jerrors)
+				default:
+					log.Error(ctx, map[string]interface{}{
+						"err": err,
+					}, "unable to list the work items")
```

Comment:
`log.Error` function must be printing the message to standard error output, right?